utils/hooks.ts

```diff
@@ -109,12 +110,23 @@ const useGeocodeQuery = () => {
           setResults([]);
         } else {
           const filteredResults = filterDuplicatePlaces(nominatimResults);
-          const formattedResults = filteredResults.map((result) => ({
-            location: new LngLat(Number(result["lon"]), Number(result["lat"])),
-            name: result["display_name"],
-            simplifiedName: simplifyPlaceDisplayName(result),
-            isRegion: !nonRegionKeys.some((k) => k in result.address),
-          }));
+          const formattedResults = filteredResults.map((result) => {
+            const firstElem: any = result["boundingbox"].shift();
+            const lastElem: any = result["boundingbox"].pop();
+            result["boundingbox"].push(firstElem);
+            result["boundingbox"].unshift(lastElem);
```

Comment:
Could you not have use a temp variable to hold the number and do a simple swap to avoid needing to use `any` and all the array changing operations?

Reply:
I used finally `as number` in the right side of the line... is it valid? 😅

Reply:
Yeah I think it's ok in this case given we know that `boundingbox` will be a tuple with 4 numbers... assuming we trust the API will always return it. If it doesn't for whatever reasons in future, the app will probably break and we will probably have to update the code anyway so I'm ok to be pragmatic with the type assertion here 👍